api/lua/pinnacle/tag.lua

```diff
@@ -194,6 +194,37 @@ function tag.remove(tags)
     end
 end
 
+---Move existing tags to the specified output.
+---
+---#### Example
+---```lua
+---local tag_to_move = Tag.get("1")
+---Tag.switch_output(Output.get_by_name("HDMI-1"), tag_to_move)
+---```
+---
+---@param output pinnacle.output.OutputHandle The output to add tags to.
+---@param ... string The names of the new tags.
+---
+---@overload fun(output: pinnacle.output.OutputHandle, tag_names: string[])
+function tag.switch_output(output, ...)
+    local tag_names = { ... }
+    if type(tag_names[1]) == "table" then
+        tag_names = tag_names[1] --[=[@as string[]]=]
+    end
```

Comment:
The type (L206) should be `TagHandle` here.

I don't think ellipsis are warranted here. Most (if not all) functions in this file use the concrete type in a single parameter (so `TagHandle[]`) instead of ellipsis, and IMO this one shouldn't be different. This will remove the need to fixup the type line 211.